[def-9] Change color active bread-crumble btn:

shaguahill opened the pull request:
Added opacity to breadcrumb-item and unset it for current page
Added hover to not active buttons
Added few changes to links, using thymeleaf and webContext

shaguahill committed:
[def-9] Change color active bread-crumble btn:
Added opacity to breadcrumb-item and unset it for current page
Added hover to not active buttons
Added few changes to links, using thymeleaf and webContext

Yuliia-Sh commented on src/main/webapp/assets/css/common.css:
```diff
@@ -127,10 +128,18 @@ main {
+    opacity: unset;
+}
 /*Bread Crumb End*/
 
```
hover тут недостаточно? для чего еще класс __active?

Rooman commented on src/main/webapp/WEB-INF/templates/all-users.html:
```diff
@@ -23,9 +23,9 @@
-                <li class="breadcrumb-item"><a href="../admin">Admin page</a></li>
-                <img class="bread__crumb-arrow" src="../assets/images/Arrow.jpg" alt="arrow">
-                <li class="breadcrumb-item active" aria-current="page">All users</li>
+                <li class="breadcrumb-item"><a th:href="@{/admin}">Admin page</a></li>
```
А можем зарефакторить этот ужастный кусок?
В списке могут быть только елементы li. 
Тег img мы используем для отображения контента страницы (фото товара, фото юзера). Декоративные имеджы вещаем на CSS. 
Соответсвенно удаляем отсюда img, и делаем стрелки через :after для li.

shaguahill replied:
Перенесено в css

shaguahill committed:
[def-9] Change color active bread-crumble btn(Review fix):
Moved arrow to css :after
Deleted opacity, because if in main class opacity=0.4, then in after opacity=0.8 == 0.4*0.8=0.32

shaguahill committed:
Merge branch 'develop' into def-9_change_color_active_bread-crumble_btn

shaguahill committed:
Merge branch 'develop' into def-9_change_color_active_bread-crumble_btn
